packages/flutter/lib/src/material/floating_action_button.dart

```diff
@@ -411,6 +423,25 @@ class FloatingActionButton extends StatelessWidget {
     final ThemeData theme = Theme.of(context);
     final FloatingActionButtonThemeData floatingActionButtonTheme = theme.floatingActionButtonTheme;
 
+    // Applications should no longer use accentIconTheme's color to configure
+    // the foreground color of floating action buttons. For more information, see
+    // flutter.dev/go/remove-fab-accent-theme-dependency.
```

Comment:
Make it a link: add `https://` in front of the URL, so it can be clicked on in IDEs.